Classes/Controller/Frontend/SetupController.php

```diff
@@ -296,7 +297,7 @@ private function getFrontendUser(): FrontendUser
         return $user;
     }
 
-    private function getFrontendUserId(): int
+    private function getFrontendUserId()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [getFrontendUserId accesses the super-global variable $GLOBALS.](https://app.codacy.com/app/robinvonberg/cf_google_authenticator/pullRequest?prid=4501904)